mediathread/assetmgr/models.py

```diff
@@ -138,8 +144,7 @@ class Asset(models.Model):
 
     # labels which determine the saving of an asset
     # in order of priority for which label is marked primary
-    # an asset must have at least one source label from this list
-    # 'url' should probably stay at the end
```

Comment:
'url' is not at the end of `primary_labels`, so I think this comment is out of date